doc/tutorial/faq.txt

```diff
@@ -0,0 +1,68 @@
+.. _faq:
```

Comment:
Also:
/u/lamblinp/code/Theano_git/doc/tutorial/faq.txt:5: WARNING: duplicate label faq, other instance in /u/lamblinp/code/Theano_git/doc/faq.txt